### PR TITLE
Id/assign pods nodes using node affinity

### PR DESCRIPTION
**Pull request by irvifa**
Related to https://github.com/kubernetes/website/issues/22296

### CODE REVIEW

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Menambahkan sebuah Label pada sebuah Node
+
+1. Jabarkan node-node yang ada pada klaster kamu, bersamaan dengan label yang ada:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:24]: "node" is an API object, it needs to be written with a capital letter.
```suggestion
1. Jabarkan Node-Node yang ada pada klaster kamu, bersamaan dengan label yang ada:
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+
+    Pada keluaran dari perintah di atas, kamu dapat melihat bahwa node `worker0`
+    memiliki label `disktype=ssd`.
+
+## Menjadwalkan Pod menggunakan Afinitas Node
+
+Konfigurasi ini menunjukkan sebuah Pod yang memiliki afinitas node `requiredDuringSchedulingIgnoredDuringExecution`, `disktype: ssd`.
+Dengan kata lain, pod hanya akan dijadwalkan hanya pada node yang memiliki label `disktype=ssd`.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:65]: "Pod" is an API object, it needs to be written with a capital letter.
```suggestion
Dengan kata lain, Pod hanya akan dijadwalkan hanya pada node yang memiliki label `disktype=ssd`.
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    ```
+    NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+    nginx    1/1       Running   0          13s    10.200.0.4   worker0
+    ```
+
+## Jadwalkan Pod menggunakan Afinitas Node yang Dipilih
+
+Konfigurasi ini memberikan deskripsi sebuah Pod yang memiliki afinitas node `preferredDuringSchedulingIgnoredDuringExecution`,`disktype: ssd`.
+Artinya pod akan diutamakan dihalankan pada node yang memiliki labe `disktype=ssd`.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:91]: typo?
```suggestion
Artinya pod akan diutamakan dijalankan pada node yang memiliki labe `disktype=ssd`.
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+    nginx    1/1       Running   0          13s    10.200.0.4   worker0
+    ```
+
+
+
+## {{% heading "whatsnext" %}}
+
+Pelajari lebih lanjut mengenai
+[Node Affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:119]: We can translate the hyperlink and link to ID Doc.
```suggestion
[Afinitas Node](/id/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    ```
+    Keluaran dari perintah tersebut akan berupa:
+
+    ```shell
+    NAME      STATUS    ROLES    AGE     VERSION        LABELS
+    worker0   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker0
+    worker1   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker1
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+1. Pilihkan salah satu dari node yang ada dan tambahkan label pada node tersebut.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:37]:
```suggestion
1. Pilihkan salah satu dari Node yang ada dan tambahkan label pada Node tersebut.
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+    ```shell
+    kubectl label nodes <your-node-name> disktype=ssd
+    ```
+    dimana `<nama-node-kamu>` merupakan nama dari node yang kamu pilih.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:42]:
```suggestion
    dimana `<nama-node-kamu>` merupakan nama dari Node yang kamu pilih.
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+1. Keluaran dari node yang kamu pilih dan sudah memiliki label `disktype=ssd`:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:44]:
```suggestion
1. Keluaran dari Node yang kamu pilih dan sudah memiliki label `disktype=ssd`:
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+    Keluaran dari perintah tersebut akan berupa:
+
+    ```
+    NAME      STATUS    ROLES    AGE     VERSION        LABELS
+    worker0   Ready     <none>   1d      v1.13.0        ...,disktype=ssd,kubernetes.io/hostname=worker0
+    worker1   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker1
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+
+    Pada keluaran dari perintah di atas, kamu dapat melihat bahwa node `worker0`
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:59]:
```suggestion
    Pada keluaran dari perintah di atas, kamu dapat melihat bahwa Node `worker0`
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+    memiliki label `disktype=ssd`.
+
+## Menjadwalkan Pod menggunakan Afinitas Node
+
+Konfigurasi ini menunjukkan sebuah Pod yang memiliki afinitas node `requiredDuringSchedulingIgnoredDuringExecution`, `disktype: ssd`.
+Dengan kata lain, Pod hanya akan dijadwalkan hanya pada node yang memiliki label `disktype=ssd`.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:65]:
```suggestion
Dengan kata lain, Pod hanya akan dijadwalkan hanya pada Node yang memiliki label `disktype=ssd`.
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+{{< codenew file="pods/pod-nginx-required-affinity.yaml" >}}
+
+1. Terapkan konfigurasi berikut untuk membuat sebuah pod yang akan dijadwalkan pada node yang kamu pilih:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:69]:
```suggestion
1. Terapkan konfigurasi berikut untuk membuat sebuah Pod yang akan dijadwalkan pada Node yang kamu pilih:
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/pods/pod-nginx-required-affinity.yaml
+    ```
+
+1. Verifikasi apakah pod yang kamu pilih sudah dijalankan pada node yang kamu pilih:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:75]:
```suggestion
1. Verifikasi apakah Pod yang kamu pilih sudah dijalankan pada Node yang kamu pilih:
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+    Keluaran dari perintah tersebut akan berupa:
+
+    ```
+    NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+    nginx    1/1       Running   0          13s    10.200.0.4   worker0
+    ```
+
+## Jadwalkan Pod menggunakan Afinitas Node yang Dipilih
+
+Konfigurasi ini memberikan deskripsi sebuah Pod yang memiliki afinitas node `preferredDuringSchedulingIgnoredDuringExecution`,`disktype: ssd`.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:90]:
```suggestion
Konfigurasi ini memberikan deskripsi sebuah Pod yang memiliki afinitas Node `preferredDuringSchedulingIgnoredDuringExecution`,`disktype: ssd`.
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+Artinya pod akan diutamakan dijalankan pada node yang memiliki labe `disktype=ssd`.
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:91]:
```suggestion
Artinya Pod akan diutamakan dijalankan pada Node yang memiliki label `disktype=ssd`.
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+
+{{< codenew file="pods/pod-nginx-preferred-affinity.yaml" >}}
+
+1. Terapkan konfigurasi berikut untuk membuat sebuah pod yang akan dijadwalkan pada node yang kamu pilih:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:95]:
```suggestion
1. Terapkan konfigurasi berikut untuk membuat sebuah Pod yang akan dijadwalkan pada Node yang kamu pilih:
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 15]
```diff
@@ -0,0 +1,119 @@
...
+
+    ```shell
+    kubectl apply -f https://k8s.io/examples/pods/pod-nginx-preferred-affinity.yaml
+    ```
+
+1. Verifikasi apakah pod yang kamu pilih sudah dijalankan pada node yang kamu pilih:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:101]:
```suggestion
1. Verifikasi apakah Pod yang kamu pilih sudah dijalankan pada Node yang kamu pilih:
```

**Comment by irvifa** [comment]
/close
In favor of https://github.com/kubernetes/website/pull/22481

**Pull request closed without merging**